package.json

```diff
@@ -1,44 +1,45 @@
 {
-  "name": "businext",
-  "version": "0.0.0",
-  "private": true,
-  "description": "Business Insight Extractor and Interpreter",
-  "main": "lib/index.js",
-  "scripts": {
-    "build": "tsc",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node lib",
-    "start:dev": "concurrently 'tsc --watch' 'nodemon lib'"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/businext/businext.ai.git"
-  },
-  "contributors": [
-    "Joshua Cheng <josh510cheng@gmail.com>",
-    "Benny Lam <bennylam6@gmail.com>",
-    "Justin Xu <xu.justin.j@gmail.com>",
-    "Tim Zwart <timrzwart@gmail.com>"
-  ],
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/businext/businext.ai/issues"
-  },
-  "dependencies": {
-    "@googlemaps/google-maps-services-js": "^3.1.16",
-    "apollo-server": "^2.22.2",
-    "class-validator": "^0.13.1",
-    "graphql": "^15.5.0",
-    "opencage-api-client": "^1.0.0",
-    "reflect-metadata": "^0.1.13",
-    "type-graphql": "^1.1.1",
-    "word2vec": "^1.1.4",
-    "yelp-fusion": "^3.0.0"
-  },
-  "devDependencies": {
-    "@types/node": "^14.14.37",
-    "concurrently": "^6.0.1",
-    "nodemon": "^2.0.7",
-    "typescript": "^4.2.4"
-  }
+	"name": "businext",
+	"version": "0.0.0",
+	"private": true,
+	"description": "Business Insight Extractor and Interpreter",
+	"main": "lib/index.js",
+	"scripts": {
+		"build": "tsc && npm run copySchema",
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"start": "node lib",
+		"start:dev": "concurrently 'tsc --watch' 'nodemon lib --ext js,graphql'",
+		"setupDockerImage": "npm install && npm run build",
```

Comment:
this command no longer exists now that we have multistage builds

rebase or merge from main to use the more updated code